pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -310,6 +310,7 @@ public void start() throws PulsarServerException {
 
     @Override
     public void disableBroker() throws Exception {
+        stop();
```

Comment:
I think `disableBroker` should just disable broker by deleting the node and it should not stop the load-manager.